edx_exams/apps/core/management/commands/test/test_bulk_add_course_staff.py

```diff
@@ -0,0 +1,96 @@
+""" Tests for bulk_add_course_staff management command """
+from tempfile import NamedTemporaryFile
+from testfixtures import LogCapture
+from unittest.mock import patch
+
+import ddt
+from django.core.management import call_command
+from django.test import TestCase
+
+from edx_exams.apps.core.models import (
+    CourseStaffRole,
+    User,
+)
+from edx_exams.apps.core.test_utils.factories import CourseStaffRoleFactory, UserFactory
+
+# todo: is this the correct logger name?
+LOGGER_NAME = 'edx_exams.apps.core.management.commands.bulk_add_course_staff'
+
+
+class TestBulkAddCourseStaff(TestCase):
```

Comment:
I think it would be worth adding a test here that ensures this command produces the expected number or queries w/ `assertNumQueries`. example: https://github.com/edx/edx-exams/blob/a6bedee649c974ac6244472293bb159249ed7e77/edx_exams/apps/lti/tests/test_views.py#L716

Reply:
That makes sense! When I used this method, I see that there are a number of calls: https://github.com/edx/edx-exams/actions/runs/8394389300/job/22991407617?pr=255. Is it better to just check relative number of queries (for example, that batch size affects the number of queries for example) or is there a better way to finetune? I couldn't figure out a good way to constrain to only a certain type of query (for example, just the bulk query). Was poking around here: https://docs.djangoproject.com/en/5.0/topics/testing/tools/

Reply:
You can probably just make this it's own test rather than baking it into the others as an extra assertion. For that one I'd try a bunch of extra rows to ensure the query count has not increased in an unexpected way based on more data.

For getting the expected number you should be able to walk though the code and count things up for what you expect. If it's different, it would be good to know why or where the extra query is coming from. There's a few ways to debug django queries but an easy one is to use

```
from django.db import connection
print(connection.queries)
```